rcljava/src/main/java/org/ros2/rcljava/RCLJava.java

```diff
@@ -255,13 +255,37 @@ public static Node createNode(final String nodeName) {
    * @return A @{link Node} that represents the underlying ROS2 node
    *     structure.
    */
+  public static Node createNode(final String nodeName, final String namespace) {
+    return createNode(nodeName, namespace, new NodeOptions());
+  }
+
+  /**
+   * Create a @{link Node}.
+   *
+   * @param nodeName The name that will identify this node in a ROS2 graph.
```

Comment:
nit:
```suggestion
   * @param nodeName The name that will identify this node in a ROS 2 graph.
```

Reply:
Replaced `ROS2` with `ROS 2` in all the file, see https://github.com/osrf/ros2_java/pull/46/commits/9fabd7ed1ba23770de63956d5a917029058f4052.